packages/sdks/src/components/block/components/interactive-element.lite.tsx

```diff
@@ -34,16 +38,18 @@ export default function InteractiveElement(
     <props.Wrapper
       {...props.wrapperProps}
       attributes={{
-        ...getBlockProperties({
-          block: props.block,
-          context: props.context.value,
-        }),
-        ...getBlockActions({
-          block: props.block,
-          rootState: props.context.value.rootState,
-          rootSetState: props.context.value.rootSetState,
-          localState: props.context.value.localState,
-          context: props.context.value.context,
+        ...(props.includeBlockProps && {
```

Comment:
You could remove one layer of spreading here:

```ts
attributes={props.includeBlockProps
  ? {
      ...getBlockProperties({
        block: props.block,
        context: props.context.value,
      }),
      ...getBlockActions({
        block: props.block,
        rootState: props.context.value.rootState,
        rootSetState: props.context.value.rootSetState,
        localState: props.context.value.localState,
        context: props.context.value.context,
      }),
    }
  : {}}
```

Reply:
Thanks a lot, let me apply this!